packages/swap/src/swappers/evm/get-execute-data.ts

```diff
@@ -0,0 +1,228 @@
+import type { Token } from '@rarimo/bridge'
+import {
+  Amount,
+  BUNDLE_SALT_BYTES,
+  isUndefined,
+  MASTER_ROUTER_ABI,
+  toLowerCase as lc,
+  TransactionBundle,
+} from '@rarimo/shared'
+import { utils } from 'ethers'
+
+import {
+  CALLER_ADDRESS,
+  CONTRACT_BALANCE,
+  THIS_ADDRESS,
+  WRAPPED_CHAIN_TOKEN_SYMBOLS as wrapped,
+} from '@/const'
+import { SwapCommands as cmds } from '@/enums'
+import type { ExecuteArgs } from '@/types'
+
+import { buildPayload as cmd } from './payload-builder'
```

Comment:
IMO using aliases for named exports without the direct need (name conflict) brings up one more level of complexity. You no more rely on usual names, instead, you need to remember the aliases for each alias which often can be not obvious.

E.g. here we have `cmds` (abbreviation for "commands", plural) for commands enum, while `cmd` ("command", singular) stands for builder function. It leads to naming inconsistency and inconvenience while reading the code